source/configure/configuration-settings.rst

```diff
@@ -4690,6 +4690,20 @@ This configuration setting is disabled by default and requires `developer mode <
 | This feature's ``config.json`` setting is ``"DeveloperFlags": ""`` with string input.  |
 +----------------------------------------------------------------------------------------+
 
+Enable API Trigger Admin Notifications
```

Comment:
When https://github.com/mattermost/docs/pull/5860 is approved and merged into production, all remaining config settings on the main config page will move to their new homes. 

Developer Flag config settings, including this new one, will soon live on the [Experimental configuration settings page](http://mattermost-docs-preview-pulls.s3-website-us-east-1.amazonaws.com/5860/configure/experimental-configuration-settings.html#developer-flags).

This page of config settings hasn't yet been migrated over to the new config settings format, so the content proposed here can remain as-is. I recommend that we merge https://github.com/mattermost/docs/pull/5860 first, then merge the latest master into this PR, then move this new config setting to its new home.